modules.d/95nfs/nfs-start-rpc.sh

```diff
@@ -17,8 +15,8 @@ if modprobe sunrpc || strstr "$(cat /proc/filesystems)" rpc_pipefs; then
     # Start rpc.statd as mount won't let us use locks on a NFSv4
     # filesystem without talking to it. NFSv4 does locks internally,
     # rpc.lockd isn't needed
-    [ -z "$(pidof rpc.statd)" ] && rpc.statd
-    [ -z "$(pidof rpc.idmapd)" ] && rpc.idmapd
+    command -v rpc.statd >/dev/null && [ -z "$(pidof rpc.statd)" ] && rpc.statd
+    command -v rpc.idmapd >/dev/null && [ -z "$(pidof rpc.idmapd)" ] && rpc.idmapd
```

Comment:
you might want `type -p` here instead of `command -v`

Reply:
Hmm, there are a few other `command -v` usage in this script file, so I used `command -v` to keep a consistent code style, any advantage of using `type -p` instead? If `type -p` is superior, perhaps I should change all `command` usage in this script to `type`.

Reply:
Also `type -p` doesn't work in `dash`